src/main/java/kr/codesqaud/cafe/config/GlobalExceptionHandler.java

```diff
@@ -17,19 +15,14 @@ public class GlobalExceptionHandler {
 
     private final Logger logger = LoggerFactory.getLogger(GlobalExceptionHandler.class);
 
-    @ExceptionHandler(BadRequestException.class)
-    public String badRequest(BadRequestException e, Model model) {
-        BindingResult bindingResult = new BeanPropertyBindingResult(e.getErrorData(), getClassName(e));
-        bindingResult.rejectValue(e.getField(), e.getErrorCode());
-        model.addAttribute(String.format("org.springframework.validation.BindingResult.%s",
-            bindingResult.getObjectName()), bindingResult);
-        model.addAttribute(bindingResult.getObjectName(), e.getErrorData());
-        return e.getViewName();
-    }
-
-    private String getClassName(BadRequestException e) {
-        String className = e.getErrorData().getClass().getSimpleName();
-        return className.replace(className.charAt(0), (char) (className.charAt(0) + 32));
+    @ResponseStatus(HttpStatus.UNAUTHORIZED)
+    @ExceptionHandler(Unauthorized.class)
```

Comment:
```suggestion
    @ExceptionHandler(UnauthorizedException.class)
```
보편적인 Java 사용자들의 Convention 상 Exception 클래스는 `Exception` 을 붙여주는게 더 좋을 것 같습니다.

Reply:
넵 바로 적용하겠습니다.